corehq/apps/users/static/users/js/roles.js

```diff
@@ -483,7 +498,9 @@ hqDefine('users/js/roles',[
         self.roleError = ko.observable("");
         self.setRoleError = function (form, error) {
             self.roleError(error);
-            $(form).find('[type="submit"]').enableButton();
+            setTimeout(() => {
+                $(form).find('[type="submit"]').enableButton();
```

Comment:
Why does this need a delay?

Reply:
It seems that if it's called directly this call precedes the call to disable the button and so it has no effect.